go/kbfs/simplefs/simplefs.go

```diff
@@ -1114,17 +1127,26 @@ func (k *SimpleFS) doCopy(
 }
 
 // SimpleFSCopy - Begin copy of file or directory
-func (k *SimpleFS) SimpleFSCopy(ctx context.Context, arg keybase1.SimpleFSCopyArg) error {
+func (k *SimpleFS) SimpleFSCopy(
+	ctx context.Context, arg keybase1.SimpleFSCopyArg) (err error) {
 	return k.startAsync(ctx, arg.OpID, keybase1.AsyncOps_COPY,
 		keybase1.NewOpDescriptionWithCopy(
 			keybase1.CopyArgs{OpID: arg.OpID, Src: arg.Src, Dest: arg.Dest}),
+		arg.IdentifyBehavior,
 		func(ctx context.Context) (err error) {
 			return k.doCopy(ctx, arg.OpID, arg.Src, arg.Dest)
 		})
 }
 
 // SimpleFSSymlink starts making a symlink of a file or directory
-func (k *SimpleFS) SimpleFSSymlink(ctx context.Context, arg keybase1.SimpleFSSymlinkArg) (err error) {
+func (k *SimpleFS) SimpleFSSymlink(
+	ctx context.Context, arg keybase1.SimpleFSSymlinkArg) (err error) {
+	if arg.IdentifyBehavior != nil {
```

Comment:
Why not pass the identify behavior into `startSyncOp` instead of handling it separately in each function?  Actually, it would be even better if it was an argument to `makeContext`, which is called in both places, and others too.